app/views/directives/build-status.html

```diff
@@ -1,12 +1,12 @@
 <status-icon status="build.status.phase" disable-animation></status-icon>
-<span ng-if="!build.status.reason || build.status.phase === 'Cancelled'">{{build.status.phase}}</span><span
-  ng-if="build.status.reason && build.status.phase !== 'Cancelled'">{{build.status.reason | sentenceCase}}</span><span
-  ng-switch="build.status.phase" class="hide-ng-leave" ng-if="build.status.startTimestamp"><span
-    ng-switch-when="Complete">, ran for {{build.status.startTimestamp | timeOnlyDurationFromTimestamps : build.status.completionTimestamp}}</span><span
-    ng-switch-when="Failed">, ran for {{build.status.startTimestamp | timeOnlyDurationFromTimestamps : build.status.completionTimestamp}}</span><span
-    ng-switch-when="Cancelled"> after {{build.status.startTimestamp | timeOnlyDurationFromTimestamps : build.status.completionTimestamp}}</span><span
-    ng-switch-when="Running"> for <time-only-duration-until-now timestamp="build.status.startTimestamp" time-only></time-only-duration-until-now></span><span
-    ng-switch-when="New"></span><span
-    ng-switch-when="Pending"></span><span
-    ng-switch-default>, ran for {{build.status.startTimestamp | duration : build.status.completionTimestamp}}</span>
+<span ng-if="!build.status.reason || build.status.phase === 'Cancelled'">{{build.status.phase}}</span>
```

Comment:
@jwforres This is only used on the monitoring page. I wanted it to be consistent with container ready for pods (see screenshot in description), and I think it's easier to scan the page without the "ran for..." It also takes less space, and we're somewhat space constrained here.

```
$ rg -thtml build-status
app/views/builds.html
105:                        <!-- <build-status build="build"></build-status> -->

app/views/monitoring.html
386:                              <build-status build="build"></build-status>
```